app/components/DisclaimerModal.tsx

```diff
@@ -0,0 +1,38 @@
+import {Text, Modal, Pressable} from 'react-native';
```

Comment:
modal looks a bit out of style

Reply:
![rev](https://github.com/user-attachments/assets/b1fb7236-8d69-4fc5-baf2-b1a52439151a)